### DISCUSSION TITLE
SWC support ?

### DISCUSSION THREAD

**Discussion by hrasoa** [facebookincubator/fbt#349] [General]
Due to the growth of [swc](https://swc.rs/), is there any plan to support it ?

**Comment by kayhadrin** [comment]
First time I'm hearing about SWC! 
It looks promising but it's not a priority for us.
Based on [this thread](https://github.com/swc-project/swc/issues/2175), there's a chance that SWC remains compatible with Babel plugins so we might not have to do anything on our side.

Please feel free to let us know if you manage to make our library work with SWC. :-)
